src/main/java/com/github/stickerifier/stickerify/logger/HighlightHelper.java

```diff
@@ -10,7 +12,7 @@
 public final class HighlightHelper {
 
 	static final String START_GREEN = changeColorTo(BOLD + GREEN_FG);
-	private static final Pattern MIME_TYPE_PATTERN = Pattern.compile(" (\\w+/[-+.\\w]+) ");
+	private static final Pattern MIME_TYPE_PATTERN = Pattern.compile("(^|\\s)(\\w+/[-+.\\w]+)(?=\\s|$)");
```

Comment:
Why did you need to change the regex? Can you please provide an example of incorrect value matched by its previous version?